functional tests don't compile on jdknext

https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_extended.functional_s390x_linux_OpenJDK/64
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_sanity.functional_s390x_linux_OpenJDK/77
https://ci.eclipse.org/openj9/job/Test_openjdknext_j9_sanity.functional_x86-64_windows_OpenJDK/63
etc. for all platforms.
```
06:49:30  compile:
06:49:30       [echo] Ant version is Apache Ant(TM) version 1.9.6 compiled on July 20 2018
06:49:30       [echo] ============COMPILER SETTINGS============
06:49:30       [echo] ===fork:                         yes
06:49:30       [echo] ===executable:                   /home/jenkins/jenkins-agent/workspace/Test_openjdknext_j9_extended.functional_s390x_linux_OpenJDK/openjdkbinary/j2sdk-image/bin/javac
06:49:30       [echo] ===debug:                        on
06:49:30       [echo] ===destdir:                      /home/jenkins/jenkins-agent/workspace/Test_openjdknext_j9_extended.functional_s390x_linux_OpenJDK/openjdk-tests/functional/cmdLineTests/jvmtitests/bin
06:49:30      [javac] Compiling 317 source files to /home/jenkins/jenkins-agent/workspace/Test_openjdknext_j9_extended.functional_s390x_linux_OpenJDK/openjdk-tests/functional/cmdLineTests/jvmtitests/bin
06:49:31      [javac] /home/jenkins/jenkins-agent/workspace/Test_openjdknext_j9_extended.functional_s390x_linux_OpenJDK/openjdk-tests/functional/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/util/SyncThread.java:63: error: invalid use of a restricted identifier 'yield'
06:49:31      [javac]         	yield();
06:49:31      [javac]         	^
06:49:31      [javac]   (to invoke a method called yield, qualify the yield with a receiver or type name)
06:49:31      [javac] /home/jenkins/jenkins-agent/workspace/Test_openjdknext_j9_extended.functional_s390x_linux_OpenJDK/openjdk-tests/functional/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/util/SyncThread.java:69: error: invalid use of a restricted identifier 'yield'
06:49:31      [javac]         yield();
06:49:31      [javac]         ^
06:49:31      [javac]   (to invoke a method called yield, qualify the yield with a receiver or type name)
06:49:31      [javac] 2 errors
```

--- a/test/functional/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/util/SyncThread.java
+++ b/test/functional/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/util/SyncThread.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2001, 2018 IBM Corp. and others
+ * Copyright (c) 2001, 2019 IBM Corp. and others
  *
  * This program and the accompanying materials are made available under
  * the terms of the Eclipse Public License 2.0 which accompanies this
@@ -60,13 +60,13 @@ public class SyncThread extends Thread
             lock.notify();
         }
         while(!isInterrupted() && work()) {
-        	yield();
+        	Thread.yield();
         }
     }
         
     protected boolean work() 
     {
-        yield();
+        Thread.yield();
         return true;
     }
 }

